projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -36,54 +44,95 @@ export function clickOnRequestQuoteInCartAndExpectQuotePage(): void {
  * Clicks on 'Request Quote' on the cart page.
  */
 export function clickOnRequestQuoteInCart(): void {
+  log(
+    'Click on "request Quote" button in the cart',
+    clickOnRequestQuoteInCart.name
+  );
   cy.get('cx-quote-request-button button').click();
 }
-
+/**
+ * Use a cx-login-form to login a user
+ * @param email Email for the login
+ * @param password Password for the login
+ * @param name Name of the user
+ */
 export function login(email: string, password: string, name: string): void {
-  // Click on the 'Sign in / Register' link
-  // & wait until the login-form is displayed
-  cy.get('cx-login [role="link"]')
-    .click()
-    .then(() => {
-      cy.get('cx-login-form').should('be.visible');
-    });
-  // Login via authentication service
-  authentication.login(email, password);
-  // Verify whether the user logged in successfully,
-  // namely the logged in user should be greeted
-  cy.get('.cx-login-greet').should('contain', name);
-  cy.get('cx-login').should('not.contain', 'Sign In');
+  trace(
+    login.name,
+    () => {
+      cy.get('cx-login [role="link"]')
+        .click()
+        .then(() => {
+          cy.get('cx-login-form').should('be.visible');
+        });
+      authentication.login(email, password);
+      cy.get('.cx-login-greet').should('contain', name);
+      cy.get('cx-login').should('not.contain', 'Sign In');
+    },
+    { email: email, password: password, name: name }
+  );
 }
 
+/**
+ * Request a quote from cart
+ * @param shopName Name of the given shop (Powertools)
+ * @param productName Name of the product that should be used for the quote
+ * @param quantity Quantity of the product used for the quote
+ */
 export function requestQuote(
   shopName,
   productName: string,
   quantity: string
 ): void {
-  this.addProductToCartForQuotePreparation(shopName, productName, quantity);
-  this.clickOnRequestQuoteInCartAndExpectQuotePage();
+  trace(
```

Comment:
Please, use your `log()` method instead of `trace()` one

Reply:
done